storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8150,7 +8150,7 @@ void ha_rocksdb::setup_scan_iterator(
 
       auto read_opts = rocksdb::ReadOptions();
       read_opts.snapshot= m_scan_it_snapshot;
-      m_scan_it= rdb->NewIterator(read_opts);
+      m_scan_it= rdb->NewIterator(read_opts, kd.get_cf());
```

Comment:
I just realized that I'm not setting the read_opts from `skip_bloom` and `fill_cache` in this case. Do you think this is necessary?